models/profile.js

```diff
@@ -0,0 +1,36 @@
+module.exports = (sequelize, DataTypes) => {
+  const Profile = sequelize.define('Profile', {
+
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+
+    imageUrl: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
```

Comment:
I don't think images should be enforced here unless the README suggests otherwise. There should be opportunity for addition or removal of profile pictures as is prevalent in a host of sites.

Hence, `allowNull` and `validate` might not be needed here.

Reply:
ok. noted